package.json

```diff
@@ -73,7 +73,7 @@
     "test": "jest",
     "lint": "eslint --ext .js,.vue ./",
     "prettify": "prettier --write '**/*.{js,json,md,scss,ts,vue}'",
-    "release": "npm run build && npm run test && standard-version",
+    "release": "npm run build && npm run test && standard-version && npm run prettify",
```

Comment:
This line doesn't work with the current rc versioning,  so each command has to be run individually in order to pass the preferred release tag as an argument to standard-version.

Reply:
Oh yeah, missed that.